### PR TITLE
Fix of servo pin on Azteeg X3 Pro to lineup with defaults

### PR DESCRIPTION
**Pull request by Natealus**
I had set mine to servo 2 because of an oddity a while ago that was
probably fixed already and seems to work just the same on Servo 0 which
lines up with default configs. By the way is there a good reason to have
the default on 3 servos I'm not aware of? I tested it out setting to
# define NUM_SERVOS 1 and it works much the same too. Almost seems like

it changes the angle a wee bit.  But is it the same difference if the
other two are -1 anyways?

### CODE REVIEW

**Comment by thinkyhead** [comment]
My guess is it's 3 because the index of the Z servo would be 2, and that's the one servo that most robots will use.

File: Marlin/ultralcd.cpp [hunk 1]
```diff
@@ -670,7 +670,7 @@ static void lcd_move_menu_axis();
 
 static void _lcd_move(const char *name, int axis, int min, int max) {
   if (encoderPosition != 0) {
-    refresh_cmd_timeout();
+    void refresh_cmd_timeout();
```
Comment (thinkyhead) [inline Marlin/ultralcd.cpp]: Wrong line. The declaration should be changed in `Marlin.h`, not this function call.

**Comment by emartinez167** [comment]
The HPX2-MAX extruder uses Servo 0 to swap extruders. As I don't have any
other servo (yet), not sure how it would work out. Also running on an
Azteeg X3 Pro.

**Comment by Natealus** [comment]
But see the other 2 are -1 and it looks like in all the default configuration.h files that its -1, -1, 0 (x, y, z)

I had it set to 2 before because I think I had an issue controlling it before but it seems to work fine now...

File: Marlin/ultralcd.cpp (hunk 1 above)
Comment (Natealus) [inline Marlin/ultralcd.cpp]: It already was from the update you pushed...still didn't work. I made this change and it fixed it.

**Comment by Natealus** [comment]
@emartinez167 do you use the same D47 that I use for your servo?

**Comment by Natealus** [comment]
If there's no particular reason to change from 3 to 1 servo cause its not really using any additional resources then might as well just leave it.

**Comment by thinkyhead** [comment]
Oops. This will work best. These lines go into `Marlin.h`…

```
extern unsigned long previous_millis_cmd;
inline void refresh_cmd_timeout() { previous_millis_cmd = millis(); }
```

Remove `static` from the declaration of `previous_millis_cmd` in `Marlin_main.cpp`…

```
unsigned long previous_millis_cmd;
```

And remove the function declaration from `Marlin_main.cpp`…

```
inline void refresh_cmd_timeout() { previous_millis_cmd = millis(); } // remove me!
```

**Comment by emartinez167** [comment]
I use 45, chosen randomly; now that I know the Viki pins, I might end up
using something else (one of the spares used by extra heaters/fans)

**Comment by Natealus** [comment]
Whoops looks like Thinky beat me to it I gotta fix this merge somehow

**Pull request closed without merging**